addons/sourcemod/scripting/saxtonhale.sp

```diff
@@ -3219,3 +3239,17 @@ stock int FindStringIndex2(int tableidx, const char[] str)
 
 	return INVALID_STRING_INDEX;
 }
+
+stock bool IsClientInRange(int iClient, float flOrigin[3], float flRadius)
+{
+	float flClientOrigin[3];
+	GetClientAbsOrigin(iClient, flClientOrigin);
+	
+	float flDistance;
+	for (int i = 0; i < sizeof(flOrigin); i++)
+	{
+		flDistance += FloatAbs(flOrigin[i] - flClientOrigin[i]);
+	}
+	
+	return flDistance <= flRadius;
+}
```

Comment:
[GetVectorDistance](https://sm.alliedmods.net/new-api/vector/GetVectorDistance) exists. Get distance from that then compare with `flRadius`. I don't mind keeping `IsClientInRange` to include `iClient` -> `flClientOrigin`